pkg/provider/gitea/acl_test.go

```diff
@@ -165,8 +179,78 @@ func TestOkToTestComment(t *testing.T) {
 				EventType:    "issue_comment",
 				Event:        issueCommentPayload,
 			},
-			allowed: false,
-			wantErr: false,
+			allowed:          false,
+			wantErr:          false,
+			rememberOkToTest: true,
+		},
+		{
+			name:          "allowed_from_org/good issue comment event without remember",
+			commentsReply: `{"body": "/ok-to-test", "user": {"login": "owner"}}`,
+			runevent: info.Event{
+				Organization: "owner",
+				Repository:   "repo",
+				Sender:       "nonowner",
+				EventType:    "issue_comment",
+				Event:        issueCommentPayload,
+			},
+			allowed:          true,
+			wantErr:          false,
+			rememberOkToTest: false,
+		},
+		{
+			name:          "allowed_from_org/good issue pull request event without remember",
+			commentsReply: `{"body": "/ok-to-test", "user": {"login": "owner"}}`,
+			runevent: info.Event{
+				Organization: "owner",
+				Repository:   "repo",
+				Sender:       "nonowner",
+				EventType:    "issue_comment",
+				Event:        pullRequestPayload,
+			},
+			allowed:          false,
+			wantErr:          false,
+			rememberOkToTest: false,
+		},
+		{
+			name:          "disallowed/bad event origin without remember",
+			commentsReply: `{"body": "/ok-to-test", "user": {"login": "owner"}}`,
+			runevent: info.Event{
+				Organization: "owner",
+				Repository:   "repo",
+				Sender:       "nonowner",
+				EventType:    "issue_comment",
+				Event:        &giteaStructs.RepositoryPayload{},
+			},
+			allowed:          false,
+			rememberOkToTest: false,
```

Comment:
In this case wantErr is false right as it goes to default case

If yes can you add `wantErr: false`

Reply:
done